applications/Colossal-LLaMA-2/train_sft.py

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env python3
```

Comment:
It seems that this file is the same as `train.py`.

If there is not difference, we can remove this file.

Reply:
Add gradient accumulation here.